lrpar/src/lib/ctbuilder.rs

```diff
@@ -97,6 +98,10 @@ where
         s.push_str(&self.stable.conflicts().unwrap().pp(&self.grm));
         s
     }
+
+    pub fn conflicts(&self) -> &STConflicts<StorageT> {
```

Comment:
Should this be `stconflicts`?

Reply:
If it's confusing, it might be worth trying a different configuration of methods to find something better. I generally find my second attempt at an API is better than the first. And I've recently sometimes had to have three attempts to find something good...

Reply:
Okay, I've tried to improve it a little bit.